### ISSUE TITLE
unexpected button text color

### ISSUE THREAD

**Issue by devonpis** [WestpacGEL/GEL-next#199]
hi, this library is awesome!
a questions about the text color on button, i try to disply the button in the most simplest way
<img width="809" alt="image" src="https://github.com/WestpacGEL/GEL-next/assets/13789367/b5fa0412-b75e-4180-81b5-13278c33b120">
but it seems the text color is always taking the value from the className 'text-text'
I assume the className `text-white` is not passing to the button
here is my code:
<img width="347" alt="image" src="https://github.com/WestpacGEL/GEL-next/assets/13789367/679b7792-d1fc-4faa-9c12-45744704147e">
do you know what is causing this?

cheers
Devon Chau

**Comment by samithaf** [comment]
@devonpis please follow the pre-requisites and installation instructions https://github.com/WestpacGEL/GEL-next#pre-requisites. 

Looks like you are not using the `withGEL` helper exported by `@westpac/ui`

```
import { withGEL } from '@westpac/ui/tailwind';
/** @type {import('tailwindcss').Config} */
const config = withGEL({
  relative: true,
  mode: 'jit',
  content: ['./src/**/*.{js,ts,jsx,tsx,mdx}', './node_modules/@westpac/ui/src/**/*.{js,ts,jsx,tsx,mdx}'],
  safelist: [],
});

export default config;
```

**Comment by devonpis** [comment]
has been resolved on @westpac/ui@0.7.1
